openedge.properties

```diff
@@ -2,12 +2,18 @@ category=Languages
 description=Code Analyzer for OpenEdge
 homepageUrl=https://github.com/Riverside-Software/sonar-openedge
 archivedVersions=
-publicVersions=2.13.1
+publicVersions=2.13.1,2.14.4
```

Comment:
You should move 2.13.1 to `archivedVersions`

Reply:
Done

Reply:
Please respond in the thread so I can refind it easily. :-)